qa/tasks/cephfs/xfstests_dev.py

```diff
@@ -61,13 +61,21 @@ def get_repos(self):
         remoteurl = 'https://git.ceph.com/xfstests-dev.git'
         self.xfstests_repo_path = self.mount_a.client_remote.mkdtemp(suffix=
                                                             'xfstests-dev')
+        # Make the xfstests_repo_path to be readable and excutable for
+        # other users, this will allow the xfstests to run the user namespace
+        # test cases, such as the generic/317.
+        self.mount_a.run_shell(['sudo', 'chmod', 'a+rx', self.xfstests_repo_path])
         self.mount_a.run_shell(['git', 'clone', remoteurl, '--depth', '1',
                                 self.xfstests_repo_path])
 
         if self.install_xfsprogs:
             remoteurl = 'https://git.ceph.com/xfsprogs-dev.git'
             self.xfsprogs_repo_path = self.mount_a.client_remote.mkdtemp(suffix=
                                                                 'xfsprogs-dev')
+            # Make the xfsprogs_repo_path to be readable and excutable for
+            # other users, this will allow the xfstests to run the user namespace
+            # test cases.
```

Comment:
both the comments are the same, maybe have just one comment to reduce verbosity? Else LGTM

Reply:
Let's keep it to make the code more easy to read.
I have updated the comments in the code as you mentioned in G-chat. @dparmar18 

Thanks.

Reply:
thanks